azure-spring-boot/src/test/java/com/microsoft/azure/keyvault/spring/CaseSensitiveKeyVaultTest.java

```diff
@@ -25,21 +23,18 @@
 
     @Test
     public void testGet() {
-        final List<String> keys = Arrays.asList("key1", "Key2");
-        
         final KeyVaultOperation keyVaultOperation = new KeyVaultOperation(
                 keyVaultClient,
-                "https:fake.vault.com",
-                TOKEN_ACQUIRE_TIMEOUT_SECS,
-                keys,
+                DEFAULT_REFRESH_INTERVAL_MS,
```

Comment:
Please change `DEFAULT_REFRESH_INTERVAL_MS` to 1 minute.

Reply:
But current value is 1800s (30 minutes), that's too long.
How about 3 minutes?

(cc @jialindai, @yiliuTo , what's your opinion?)

Reply:
What is the use.case necessitating this huge change of the refresh interval?

Reply:
Out customer may update key-value pair in key-vault, then get the pair by `environment.getProperty()` minutes latter. if customer can not get the updated data, it may make our customer confused.

Reply:
I am inclined to change the refresh value to 0 and document that the PropertySource only accesses the KeyVault upon startup and IF the customer wants to use the @Environment to get new values out of the KeyVault they need to set it according to their requirements.

Reply:
> I am inclined to change the refresh value to 0 and document that the PropertySource only accesses the KeyVault upon startup

Sounds good. I have the same idea before.